simulations/uniswap/postprocessing.py

```diff
@@ -0,0 +1,25 @@
+import os
+from functools import reduce
+from typing import List, Tuple
+
+import pandas as pd
+
+
+def save(batch_results: List[List[Tuple[int, int]]]):
```

Comment:
Is this function the same as the aave version (apart from the path), could be consolidated into one?

Reply:
Done in a new commit